services/painting-process-equipment-simulator-service/tests/test_logger.py

```
@@ -0,0 +1,324 @@
+import json
+import os
+import tempfile
+import pytest
+from unittest.mock import patch, mock_open, call, ANY
```

Ruff (0.12.2)

`unittest.mock.ANY` imported but unused

Remove unused import: `unittest.mock.ANY`

(F401)